assembly/assembly.xml

```diff
@@ -0,0 +1,77 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+
+  <id>assembly</id>
+
+  <formats>
+    <format>zip</format>
+    <format>tar.gz</format>
+  </formats>
+
+  <includeBaseDirectory>false</includeBaseDirectory>
+
+  <dependencySets>
+    <dependencySet>
+      <outputDirectory>lib</outputDirectory>
+      <useProjectArtifact>false</useProjectArtifact>
+      <useTransitiveDependencies>false</useTransitiveDependencies>
+      <excludes>
+        <exclude>io.camunda:c7-data-migrator-distro</exclude>
+        <exclude>com.h2database:h2</exclude>
+      </excludes> 
+    </dependencySet>
+    <dependencySet>
+      <outputDirectory>userlib/</outputDirectory>
+      <includes>
+        <include>com.h2database:h2</include>
+      </includes>
+    </dependencySet>
+  </dependencySets>
+
+   <files>
+  <file>
+    <source>${project.build.directory}/dependency/c7-data-migrator-distro-${project.version}.jar</source>
+    <outputDirectory>lib</outputDirectory>
```

Comment:
/suggest Can we name this folder `internal` to make it transparent that the user shouldn't mess with what's inside here? Libs are included in the Uber jar anyway, so the name `lib` is even a bit misleading, and users might be confused by the purpose of `lib` and `userlib' folders.

Reply:
Good point, updated. Changes https://github.com/camunda/c7-data-migrator/pull/15/commits/98a60eaec48eb84dd14fdc60f12230f017dc2261